test/envelope_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class TestEnvelope < MiniTest::Test
+  def setup
+    @long_note = MusicTheory::Note.new(duration: 4.0, envelope: true, output_file_name: 'delete_me')
+  end
+
+  def test_note_plays
+    @long_note.play
```

Comment:
this is an experiment, not a test...  it seemed like an easier way to experiment with immediate changes in a branch rather than merge bad changes and do the whole `bundle update` process